java-kubectl/README.md

```diff
@@ -0,0 +1,114 @@
+# Payroll
+
+This guides shows an example of developing a Spring Boot Java application with cnd. It's based on [Spring's gs-rest-service example](https://github.com/spring-guides/gs-rest-service).
+
+## Deploy the payroll service
+
+Clone the repo and move to this example folder.
+
+```console
+git clone https://github.com/okteto/examples
+cd examples/java-kubectl
+```
+
+Deploy the Payroll application by using the following command:
+```console
+kubectl apply -f manifests
+```
+
+Wait for one or two minutes until the application is running. Run the following command to get the External IP of the service:
+```console
+kubectl get service payroll
+```
+
+## Cloud native development
+
+In order to activate your Cloud Native Development, execute:
+
+```console
+cd payroll
+cnd up
+```
+
+This will create a remote container which is synchronized with your local code changes. The container already includes java dev tools (e.g. gradle), and it's configured to  automatically compile your code directly on the cluster everytime you make a change.
+
+On a second terminal screen execute the command below. This will run the `boot` command as defined on [`cnd.yml`](cnd.yml). The boot command will run `gradle bootRun` directly in the cluster. The command will start your service and reload it automatically after every succesful compilation.
```

Comment:
I'd say "boot **script**" instead of "command", it might be confused with a CND command rather than a user defined script.